docs/site/src/app/pages/docs-layout/docs-layout.component.ts

```diff
@@ -10,6 +10,8 @@ import {DocService} from "../../services/doc.service";
   styleUrls: ["./docs-layout.component.scss"]
 })
 export class DocsLayoutComponent implements OnInit {
+  innerWidth: number;
+
```

Comment:
This variable is unused I guess. If not, please migrate the dependent-logic to flex layout instead.